e2e/manifests/values/deployment-scheduler.yaml

```diff
@@ -81,7 +43,7 @@ storageClasses:
       volumeBindingMode: WaitForFirstConsumer
       allowVolumeExpansion: true
       additionalParameters:
-        topolvm.cybozu.com/device-class: "ssd"
+        topolvm.cybozu.com/device-class: "ssd"        
```

Comment:
Please remove trailing spaces.

Reply:
Done, I found a couple more in `psp.yaml` files